### DISCUSSION TITLE
Building oculus quest content in Unity using open brush assets

### DISCUSSION THREAD

**Discussion by vaidehibhargava** [icosa-gallery/open-brush#119] [Q&A] [answered]
I am making a VR experience in unity and running it on my oculus quest 2 

I used this tutorial to set it up: https://www.youtube.com/watch?v=wnn-dzHz-tA&t=766s

And then using the tilt brush sdk, I imported assets into it-

3 in total 

However one of the assets has been made grabbable -Using the instructions in this tutorial (https://www.youtube.com/watch?v=1VC3ZOxn2Lo&t=335s)

And although I can see it in the scene I can’t see it when I run it on the oculus quest

When I make it grabbable it vanishes and is visible otherwise 

Thanks a lot for your time!

**Comment by andybak** [comment, marked as answer]
> However one of the assets has been made grabbable

Does the problem go away if you remove or disable the grabbable component?

I haven't watched the video. Is the grabable component one that the tutorial author created? Is that the only component you add (well - probably that and rigidbody?)

Actually - that gives me an idea. The minute you add rigidbody, by default the object will be subject to gravity and start falling, Have you forgotten to untick "use gravity" or tick "is kinematic"?

**Reply by vaidehibhargava** [reply]
Awesome I will try this as I saw it slide out of sight

**Reply by vaidehibhargava** [reply]
I untick gravity AND tick kinematic? Thanks!

**Reply by andybak** [reply]
Depends on what the tutorial says. There's reasons to do either or both.

**Reply by vaidehibhargava** [reply]
Its just now it doesn’t seem to respond to my controllers, whereas before even tho it was not there, my controllers were transformed into hands 

I don’t think he made the component 

And he just shows you how to pick up a cube 

Does not mention ticking boxes 

I will try all the variations! 
Thanks so much!!

**Reply by vaidehibhargava** [reply]
I can’t pick it up even with both boxes unticked or just kinematic ticked 

However I am happy that I can now see it 

I will ask the tutor how to make the asset grabbable 

Or else try and find another way to make it grabbable 

It seems more to do with the technique rather than the asset 

Thanks so much for your help!!
